src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -45,7 +47,7 @@ public ErrorLogger(Stream stream, string toolName, string toolFileVersion, Versi
 
         private void WriteToolInfo(string name, string fileVersion, Version assemblyVersion)
         {
-            _writer.WriteObjectStart("toolInfo");
+            _writer.WriteObjectStart("tool");
             _writer.Write("name", name);
             _writer.Write("version", assemblyVersion.ToString(fieldCount: 3));
```

Comment:
ok, to close. Nick will update version emitted to conform to my guidance (we have tracking items to update the SARIF spec). we are adding a 'sarifLoggerVersion' member that can be populated if the sarif-producing component is versioned separately from the dependent tool. Nick doesn't need to be concerned about populating this item, as the Roslyn version uniquely identifies the sarif-producing code.